debian/mir-test-tools.install

```diff
@@ -1,6 +1,8 @@
 usr/bin/mir_stress
 usr/bin/mir_performance_tests
 usr/bin/mir-smoke-test-runner
+usr/bin/mir_platform_graphics_test_harness
```

Comment:
Isn't this line already on master?

Reply:
Yes, but that commit should be on master, too. I wonder why it's showing here again?